tools/azure-devtools/src/azure_devtools/perfstress_tests/_batch_perf_test.py

```diff
@@ -15,6 +16,10 @@
 from ._policies import PerfTestProxyPolicy
 
 
+PSTATS_PRINT_DEFAULT_SORT_KEY = pstats.SortKey.TIME
+PSTATS_PRINT_DEFAULT_LINE_COUNT = 36
```

Comment:
Is this chosen at random? Or was it in line with profile support in other languages?

Reply:
This was chosen pretty arbitarily. In my testing, typical output was 400+ lines in from cProfile/pstats, so I thought 36 was enough to just give a glimpse into the performance heavy hitters for users just viewing a pipeline run without having to download/extract/run through a parser. 

From what I can tell, no other languages have stdout output for profiling and simply output profiling data to a file path.